packages/icons/src/index.ts

```diff
@@ -24,7 +24,8 @@ if (typeof window !== 'undefined') {
   }
 
   // Defining clr-icon custom element
-  customElements.define('clr-icon', ClarityIconElement);
+  const clarityIconElementFn: unknown = ClarityIconElement;
```

Comment:
Why do we need `unknown` type here?

Reply:
eslint was complaining about the ClarityIconElement type used in customElements.define fn. The other option I could think of was to use ts-ignore or any. I went with unknown.